workload_generator/src/workload_gen.py

```diff
@@ -202,41 +209,19 @@ def performRequest(self, pid, transactionNumber, command, user=None, stockSymbol
             request['amount'] = amount
         if filename:
             request['filename'] = filename
-                
+
+        requestString = serverURL + reqType
+        
         try:
-            packetQ.put(request)
+            r = requests.post(requestString, json=command_dict)
```

Comment:
send python request and json dictionary to the django backend. 


https://pypi.org/project/requests/#description